create-publication-from-scopus/src/main/java/no/sikt/nva/scopus/ScopusConverter.java

```diff
@@ -61,9 +65,39 @@ private EntityDescription generateEntityDescription() {
         entityDescription.setReference(generateReference());
         entityDescription.setMainTitle(extractMainTitle());
         entityDescription.setContributors(generateContributors());
+        entityDescription.setTags(generatePlainTextTags());
```

Comment:
Is this the correct place to store the keywords?

Reply:
Yes, if my previous comment got deleted